src/core/packet.c

```diff
@@ -621,7 +621,7 @@ QuicPacketLogHeader(
                 (const QUIC_LONG_HEADER_V1 * const)Packet;
 
             QUIC_VAR_INT TokenLength;
-            QUIC_VAR_INT Length;
+            QUIC_VAR_INT Length = 0;
```

Comment:
Why?

Reply:
I got a compiler error in a different branch actually;  you might review the code in this area?    I'm pretty confident it was linux (not mac/windows) - but I dont remember the compiler/distro

Reply:
That's weird. We have a lot of different builds onboarded to CI and they all pass without this. Since this is in logging code, I don't really care much either way, so you can just keep it.